packages/valory/skills/termination_abci/behaviours.py

```diff
@@ -43,7 +43,7 @@
     AsyncBehaviour,
     BaseBehaviour,
 )
-
+from packages.valory.skills.transaction_settlement_abci.payload_tools import hash_payload_to_hex
```

Comment:
At some point this should be split into rounds and behaviours. But I'd do it in a separate PR for easier diffs.

Also, you're missing the definition of the AbciApp